docs/data-sources/inspection_results.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: "Inspection Results Data Source"
+Description: |-
+    Get inspection results
+---
+
+# Inspection Results Data Source
```

Comment:
I get that inspection resource doesn't fall into terraform use case, can we please capture the same in the documentation?

Reply:
Since its a datasource its evident its only for read operations and not for creation hence need not to capture it here.